frontend/lib/src/components/shared/FullScreenWrapper/FullScreenWrapper.test.tsx

```diff
@@ -51,4 +55,12 @@ describe("FullScreenWrapper", () => {
     // queryBy returns null vs. error
     expect(screen.queryByTestId("StyledFullScreenButton")).not.toBeNull() // eslint-disable-line testing-library/prefer-presence-queries
   })
+
+  it("hide find StyledFullScreenButton when hideFullScreenButton is true", () => {
+    const props = getProps({ hideFullScreenButton: true })
+
+    customRenderLibContext(<FullScreenWrapper {...props} />, {})
+    // queryBy returns null vs. error
+    expect(screen.queryByTestId("StyledFullScreenButton")).toBeNull() // eslint-disable-line testing-library/prefer-presence-queries
```

Comment:
super nit: prefer
`expect(screen.queryByTestId("StyledFullScreenButton")).not.toBeInDocument()`. That's the way Maya and I have been doing these for RTL and would be great to have this be the same.

Reply:
This indeed seems better 👍  Updated